Dockerfile

```diff
@@ -62,6 +62,20 @@ COPY config config
 COPY devtools devtools
 RUN npm install
 
+# Fetch external Bazel artifacts.
+# Copy over the WORKSPACE file, everything it imports, and bazelisk.
+COPY tools/bazelisk* tools/
+COPY build_defs/emscripten build_defs/emscripten
+COPY build_defs/kotlin_native build_defs/kotlin_native
+COPY emscripten_cache emscripten_cache
+COPY .bazelignore \
+     .bazelversion \
+     WORKSPACE \
+     BUILD.bazel \
+     maven_install.json \
+     ./
+RUN ./tools/bazelisk fetch @maven//...
```

Comment:
So, we're only fetching Mavens stuff, is that correct?

Reply:
Yes. Given that you have made the changes to only download platform-specific files, we could fetch other artifacts as well and see if the docker image is still reasonable.

Reply:
Removed the unnecessary COPY statements.